### DISCUSSION TITLE
Whisper latency SLA

### DISCUSSION THREAD

**Discussion by chaitanya1618** [deepgram/community#790] [General help] [answered]
Hi, is there an SLA for response times for whisper transcription? We are seeing response times vary wildly by even minutes on some occasions. We are seeing that in many cases, Deepgram's Whisper is more accurate than OpenAI's, but slow response times is a mjor blocker for us to switch over. Thank you!

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jpvajda** [comment, marked as answer]
@chaitanya1618  We have no SLAs for our Whisper Models in terms of guaranteeing response times.

*  It's important to understand that Whisper models are less scalable than all other Deepgram models due to their inherent model architecture. 
* Deepgram's non-Whisper models will return results faster and scale to a higher load, so we recommend using a Deepgram model such as [Nova-2](https://developers.deepgram.com/docs/models-languages-overview#nova-2) if it can meet your needs.
*  There is a 10 minute time out for all Deepgram models. Transcription requests that run longer than 10 minutes will return a 504 error.

**Comment by chaitanya1618** [comment]
Thank you!
